Week3/solution-3.txt

```diff
@@ -0,0 +1,168 @@
+Write a python script to perform CRUD operation on following table of "ESM.db" database.
+
+Employee ( eid, ename, dept, basic, branch). eid must be primary key.
+
+Department : Account, Inventory, IT, HR.
+
+Peform Following Operation.
+
+1. Create table
+
+import sqlite3 as sqlite
+con=sqlite.connect("c:\\sqlite3\\esm.db")
+q="create table emp(eid int,ename text,dept text,basic int,branch text)"
+cur=con.cursor()
+cur.execute(q)
+con.commit()
+
+
+2. Insert 5 Records directly, 5 records using tuple and 5 records using taking input from user
+
+import sqlite3 as sqlite
+con=sqlite.connect("c:\\sqlite3\\esm.db")
+cur=con.cursor()
+ins="""insert into emp values(1,'sai','it',35000,'bardoli'),
+                            (2,'ram','account',45000,'surat'),
+                            (3,'krishna','hr',30000,'baroda'),
+                            (4,'om','hr',28000,'vyara'),
+                            (5,'gopal','it',38000,'surat')"""
+cur.execute(ins)
+con.commit()
+#record input using tuple
+r=[(6,'Arpi','hr',25000,'bardoli'),
+   (7,'Astha','it',35000,'mumbai'),
+   (8,'Jiswa','account',38000,'surat'),
+   (9,'Priya','it',30000,'surat'),
+   (10,'Riddhi','inventory',28000,'navsari')]
+cur.executemany("insert into emp values(?,?,?,?,?)",r)
+con.commit()
+#record insert using user input
+ins2="insert into emp values(?,?,?,?,?)"
+l=[]
+for i in range(0,5):

```

Comment:
range() is compulsory to use ? 
if i don't use , then how i perform this query ?

Reply:
I think it's depend on a query that you are use or not